unity-client/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/AssetBundles/AB/AssetBundlesLoader.cs

```diff
@@ -8,7 +8,7 @@ namespace DCL
 {
     public class AssetBundlesLoader
     {
-        private const float MAX_LOAD_BUDGET_TIME = 0.004f;
+        private const float MAX_LOAD_BUDGET_TIME = 0.05f;
```

Comment:
This budget is too high (50ms). If too many assets are in the queue hiccups will trigger for sure.